src/uct/tcp/tcp_iface.c

```diff
@@ -448,22 +462,22 @@ static UCS_CLASS_INIT_FUNC(uct_tcp_iface_t, uct_md_h md, uct_worker_h worker,
         goto err_cleanup_rx_mpool;
     }
 
-    self->epfd = epoll_create(1);
-    if (self->epfd < 0) {
-        ucs_error("epoll_create() failed: %m");
+    status = ucs_event_set_create(&self->event_set);
+    if (status != UCS_OK) {
+        ucs_error("ucs_event_set_create() failed: %m");
```

Comment:
pls, remove the logging an error here

and add logging an error in case out-of-memory inside `ucs_event_set_create` instead

Reply:
Removed

Reply:
it seems error logging wasn't removed from here

Reply:
@hiroyuki-sato